skbio/io/tests/test_fasta.py

```diff
@@ -154,8 +178,19 @@ def setUp(self):
                  description='detailed description \t\twith  new  lines',
                  quality=[42, 42, 442, 442, 42, 42, 42, 42, 42, 43])],
             {},
-            list(map(get_data_path, ['fasta_multi_seq', 'fasta_max_width_5'])),
-            list(map(get_data_path, ['qual_multi_seq', 'qual_max_width_5']))
+            list(map(get_data_path, ['fasta_multi_seq', 'fasta_max_width_5',
+                                     'fasta_blank_lines_between_records',
+                                     'fasta_ws_lines_between_records'])),
```

Comment:
@jairideout are you sure you don't want me to add the fasta files? There's a comment in the code [here](https://github.com/anderspitman/scikit-bio/blob/3d4db981a17ea0a9d783bbabfa880c5641f30caf/skbio/io/tests/test_fasta.py#L232) which mentions that some of the test logic is intentionally duplicated even though FASTA and QUAL share the same parsing data, just in case they diverge at some point.